docs/Users_Guide/appendixH.rst

```diff
@@ -0,0 +1,87 @@
+
+.. _table_ugrid_metadata:
+
+.. list-table:: Required UGRID Metadata
+  :widths: auto
+  :header-rows: 1
+
+  * - Metadata Map Key
+    - Description
+    - Required
+  * - dim_face
+    - Dimension name for UGRID cells
+    - Required
+  * - lat_face
+    - Coordinate variable name for the latitude of each UGRID cell
+    - Required
+  * - lon_face
+    - Coordinate variable name for the longitude of each UGRID cell
+    - Required
+  * - dim_time
+    - Dimension name for the time coordinate
+    - Required
+  * - time
+    - Coordinate variable for time
+    - Required
+  * - dim_vert
+    - Dimension name for the vertical coordinate, if present
+    - Optional
+  * - vert_face
+    - Coordinate variable name for the vertical coordinate
+    - Optional
+
+Unstructured Grid Configuration Entries
+=======================================
+
+In the PointStat and GridStat config files, a user can control aspects of the UGRID capability. Since the UGRID support in MET is optional, these items are not included in the default MET config files. If the user requires modification for any of these, they must add them to their MET config file.
+
+ugrid_dataset
+-------------
+
+If the UGRID dataset is supported by MET, then this item is set to the string identifying the UGRID. Currently supported ugrid_dataset options include “lfric”, and “mpas”. If this item is not set, the user must provide a separate UGRID configuration file on the command line using the **-ugrid_config** command line argument.
+
+ugrid_max_distance_km
+---------------------
+
+For PointStat, this is the distance from each UGRID cell center that PointStat will search outward to collect observations to use for verification. The default is 0 km, which by default will use the closest observation to the UGRID cell for verification. For GridStat, this is the distance from each UGRID forecast cell center to search for observation grid cells to include when interpolating to the UGRID forecast cell locations. Currently, only the nearest point observation or gridded observation cell is used.
+
+ugrid_coordinates_file
+----------------------
+
+The absolute path to the ancillary NetCDF file that describes the UGRID topology. This file is checked for the required UGRID metadata first, and the data file is checked second and takes precedence over this file if the same elements are found in both files.
+
+ugrid_metadata_map
+------------------
+
+The mapping dictionary which allows a user to specify the variable names of the required UGRID topology elements that are required by MET. For "lfric" and "mpas", **ugrid_metadata_map** comes pre-configured with MET and a user can simply set **ugrid_dataset**. Otherwise, the user must create a separate UGRID configuration file containing **ugrid_metadata_map** and provide it on the command line using the **-ugrid_config** command line argument.
```

Comment:
Suggest referencing the talble _table_ugrid_metadata here somewhere. Maybe something like: "Otherwise, the user must create a separate UGRID configuration file containing **ugrid_metadata_map** and **the elements from Table ##.# and** provide it...."